format_spec/fragment.md

```diff
@@ -257,8 +257,8 @@ The footer is a simple blob \(i.e., _not a generic tile_\) with the following in
 | Tile null counts offset for attribute/dimension N | `uint64_t` | The offset to the generic tile storing the tile null counts for attribute/dimension N |
 | Fragment min max sum null count offset | `uint64_t` | The offset to the generic tile storing the fragment min max sum null count data. |
 | Processed conditions offset | `uint64_t` | The offset to the generic tile storing the processed conditions. |
-| Array schema name size | `uint64_t` | The total number of characters of the array schema name. |
-| Array schema name | `uint8_t[]` | The array schema name. |
```

Comment:
I removed these because array schema name and size also appear above on lines 221 and 222, I think these were misplaced in the docs?